relayer/listener.go

```diff
@@ -156,30 +156,30 @@ func NewRelayer(
 		// Process historical blocks in a separate goroutine so that the main processing loop can
 		// start processing new blocks as soon as possible. Otherwise, it's possible for
 		// ProcessFromHeight to overload the message queue and cause a deadlock.
-		go sub.ProcessFromHeight(big.NewInt(0).SetUint64(height), r.catchUpResultChan)
+		go sub.ProcessFromHeight(big.NewInt(0).SetUint64(height), lstnr.catchUpResultChan)
 	} else {
-		r.logger.Info(
+		lstnr.logger.Info(
 			"processed-missed-blocks set to false, starting processing from chain head",
-			zap.String("blockchainID", r.sourceBlockchain.GetBlockchainID().String()),
+			zap.String("blockchainID", lstnr.sourceBlockchain.GetBlockchainID().String()),
 		)
-		err = r.setAllProcessedBlockHeightsToLatest()
+		err = lstnr.setAllProcessedBlockHeightsToLatest()
 		if err != nil {
 			logger.Error(
 				"Failed to update latest processed block. Continuing to normal relaying operation",
-				zap.String("blockchainID", r.sourceBlockchain.GetBlockchainID().String()),
+				zap.String("blockchainID", lstnr.sourceBlockchain.GetBlockchainID().String()),
 				zap.Error(err),
 			)
 			return nil, err
 		}
-		r.catchUpResultChan <- true
+		lstnr.catchUpResultChan <- true
 	}
 
-	return &r, nil
+	return &lstnr, nil
 }
 
-func (r *Relayer) calculateListenerStartingBlockHeight(processHistoricalBlocksFromHeight uint64) (uint64, error) {
+func (lstnr *Listener) calculateListenerStartingBlockHeight(processHistoricalBlocksFromHeight uint64) (uint64, error) {
```

Comment:
Nit: Can we rename this `calculateStartingBlockHeight` since the `Listener` is applied in the name now?